libs/transfer/src/lib/model/tracker-events.ts

```diff
@@ -0,0 +1,5 @@
+import { UserActionTrackerEvent } from '@backbase/foundation-ang/observability';
+
+export class TransferSubmitEvent extends UserActionTrackerEvent {
+  readonly name = 'submit_transfer';
+}
```

Comment:
Don't forget to add created class to `index.ts` to make them a part of the public API